Mergy_Hand_Pose_Estimation_Preprocessing/preprocessing_module.py

```diff
@@ -0,0 +1,202 @@
+#시각화 및 엑셀 저장을 위한 모듈
+from tkinter import Y
+import openpyxl
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import pandas as pd
+import cv2
+import math
+
+class Preprocessing:
+    def __init__(self):
+        #시각화 그래프 세팅
+        """
+        plt.ion()#실시간 업데이트 허용
+        self.fig = plt.figure()
+        self.ax = self.fig.add_subplot(111)#1개의 plot인 ax를 (1,1)지점에 구현
+        plt.xlim(0,640)
+        plt.ylim(0,480)
+        plt.axis('square')
+        """
+        #3d
+        #fig = plt.figure()
+        #ax = fig.add_subplot(111, projection='3d')
+        
+        #camera input
+        self.result_counter = 0  # 결과 파일명에 사용될 카운터 변수 초기화
+        self.image_width = 640
+        self.image_height = 480
+        self.excel_path = "/home/ubuntu/intel-project/new_mergy/landmarks.xlsx"
+
+                #추출한 Point들을 xlxs에 저장하기 위한 기본 세팅
+        self.workbook = openpyxl.Workbook()
+        self.sheet = self.workbook.active
+        self.sheet.cell(row=1, column=2).value = "x"
+        self.sheet.cell(row=1, column=3).value = "y"
+        self.sheet.cell(row=1, column=4).value = "z"
+        self.x=[]
+        self.y=[]
+        self.z=[]
+        for i in range(0,13,1):
+            self.x.append(0)
+            self.y.append(0)
+            self.z.append(0)
+        
+        self.top_pad = 50
+        self.bottom_pad = 50
+        self.left_pad = 50
+        self.right_pad = 50
+        
+    
+
+
+    def getNowimage(self):
+        # 엑셀 파일에서 데이터 읽기
```

Comment:
I think also "now" keyword is weird a little.

Reply:
umm I'll think about it some more